.github/workflows/newclient.yaml

```diff
@@ -0,0 +1,60 @@
+name: New Client
+on:
+  schedule:
+    - cron: '0 2 * * *' # nightly at 2 am UTC
+  push:
+    branches: [ main ]
+    paths:
+      - generation/**
+  pull_request:
+    branches: [ main ]
+    paths:
+      - generation/**
+
+  workflow_dispatch:
+
+jobs:
+  generate:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+          cache: 'pip' # caching pip dependencies
+      - name: Get current date
+        id: date
+        run: echo "::set-output name=date::$(date +'%Y-%m-%d' --utc)"
+      - uses: actions/setup-java@v3
+        with:
+          distribution: zulu
+          java-version: 11
+      - run: java -version
+      - uses: actions/cache@v3
+        id: mvn-cache
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-unified-${{ steps.date.outputs.date }}
+      - name: Install dependencies
+        run: pip install -r generation/new_client/requirements.txt
+      - name: Generate
+        run: |
+          python generation/new_client/new-client.py generate \
```

Comment:
Can you explicitly remove java-apikeys module before new-client.py?

Reply:
I do that in new-client.py. Is that not enough?

Reply:
For me it's itchy that if, by any chahce, new-client.py does nothing, the verification in this check still passes.

Reply:
It's done below:

```
      - name: Remove Client
        run: |
          rm -rf java-${API_SHORT_NAME} || true
          sed -i "/${API_SHORT_NAME}/d" pom.xml
          sed -i "/${API_SHORT_NAME}/d" gapic-libraries-bom/pom.xml
``